pyfr/backends/cuda/blasext.py

```diff
@@ -88,6 +90,9 @@ def reduction(self, *rs, method, norm, dt_mat=None):
         # Build the reduction kernel
         rkern = self._build_kernel('reduction', src, argt)
 
+        # Configure L1
+        rkern.set_cache_pref(prefer_l1=True)
```

Comment:
As this is a reduction kernel which does use some shared memory is it worth declaring a preference?